scripts/ingests/BONES_archive/ingest_BONES_spectral_types.py

```diff
@@ -117,7 +117,10 @@ def extractADS(link):
         logger.warning(msg)
         if "Spectral type already in the database" in str(e):
             already_exists+=1
-        skipped+=1
+        else:
+            skipped += 1
+            raise e
```

Comment:
This is good -- you are expecting some to be already in the database, BUT there might be some other types of exceptions. When you encounter those, the exception should be raised, in this case, with `raise e`